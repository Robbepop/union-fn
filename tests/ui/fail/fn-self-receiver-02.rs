fn main() {}

#[union_fn::union_fn]
trait Foo {
    fn foo(&mut self) {}
}
