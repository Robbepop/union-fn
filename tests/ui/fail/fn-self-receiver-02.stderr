error: must not have self receiver argument
 --> tests/ui/fail/fn-self-receiver-02.rs:5:12
  |
5 |     fn foo(&mut self) {}
  |            ^^^^^^^^^
